text/diff_util/constants.rs

```diff
@@ -1,8 +1,8 @@
 #![allow(dead_code)]
 
-pub const EXIT_STATUS_NO_DIFFERENCE: i32 = 0;
-pub const EXIT_STATUS_DIFFERENCE: i32 = 1;
-pub const EXIT_STATUS_TROUBLE: i32 = 2;
+pub const EXIT_STATUS_NO_DIFFERENCE: u8 = 0_u8;
+pub const EXIT_STATUS_DIFFERENCE: u8 = 1_u8;
+pub const EXIT_STATUS_TROUBLE: u8 = 2_u8;
```

Comment:
Don't add type tags where the type is specified in the same line.   Just creates visual redundancies, and the redundancies do not improve code generation.

Reply:
While I was at it, I switched the tests to use these constants instead of the enum, since that seemed a bit circular.